include/exadg/acoustic_conservation_equations/spatial_discretization/spatial_operator.h

```diff
@@ -241,6 +242,11 @@ class SpatialOperator : public Interface::SpatialOperator<Number>
   InverseMassOperator<dim, 1, Number>   inverse_mass_pressure;
   InverseMassOperator<dim, dim, Number> inverse_mass_velocity;
 
+  /*
+   * RHS operator
+   */
```

Comment:
More information here, e.g. explaining `1`, similar to comment https://github.com/exadg/exadg/pull/619/files#r1426712232 would be helpful here.